interfaces/Portal/src/app/components/program-card/program-card.component.html

```diff
@@ -29,8 +29,9 @@ <h1>
 
 <ion-card
   *ngIf="program"
-  [routerLink]="['/program', program.id, program.phase]"
+  [routerLink]="['/program', program.id, 'people-affected']"
```

Comment:
Can we add this to the `AppRouts`-enum set, instead of using this magic-string?

I also think the `ProgramPhase.peopleAffected`-thing should not be used (but might be good-for-now, as it isn't really a "Program Phase"...